reg/drone/service/battery/Parameters.0.2.uavcan

```diff
@@ -0,0 +1,57 @@
+# Smart battery parameter message. It is mostly intended for automated battery charging and maintenance systems.
+# This message is modeled after the Smart Battery Data Specification (SBS) and the MAVLink battery status messages.
+#
+# The values carried by this message are either constant or slow-changing, so, generally, the publishing frequency
+# should not be higher than 0.2 Hz, and the priority should be either OPTIONAL or SLOW.
+#
+# All parameters are required unless specifically stated otherwise.
+
+truncated uint64 unique_id
+# A statistically unique number that can be used to differentiate this exact battery from others.
+# Needed for logging purposes. If the battery supports SBS, this value may be computed as CRC64WE of
+# (ManufacturerName + DeviceName + ManufactureDate + SerialNumber).
+# This value shall be invariant to the identity of the reporting node unless it is an integral part of the battery.
+# Vendors may use a constant random number for the 40 most significant bits for vendor identification purposes.
+
+uavcan.si.unit.mass.Scalar.1.0 mass
+# The total mass of the battery, including the packaging, electronics, cabling, and all auxiliary items, if any.
+# May be used for predicting the kinematic parameters of the vehicle.
+# NaN if unknown.
+
+uavcan.si.unit.electric_charge.Scalar.1.0 design_capacity
+# The maximum total charge of the pack, at 100% SoH, specified by the manufacturer.
+
+uavcan.si.unit.voltage.Scalar.1.0[2] design_cell_voltage_min_max
+# The minimum (end of discharge) and the maximum (end of charge) cell voltages specified by the manufacturer
+# at 100% SoH. The cell voltages are measured when they have stabilized after discharge/charge has been terminated. Example: {2.8, 4.2} V
+# Voltage below the min may be observed during discharge due to the cell's internal resistance. Voltage above the max voltage may
+# be observed during regenerative braking/ charging etc due to the cell's internal resistance.
+
+uavcan.si.unit.electric_current.Scalar.1.0 discharge_current            # Recommended continuous discharge current of the battery
+uavcan.si.unit.electric_current.Scalar.1.0 discharge_current_burst      # Maximum safety current that may be discharged at least for 5 seconds.
+uavcan.si.unit.electric_current.Scalar.1.0 charge_current               # Recommended continuous charge current of the battery
+uavcan.si.unit.electric_current.Scalar.1.0 charge_current_fast          # Recommended safest highest continuous charge current for the battery. This 
+                                                                        # may be expected to result in accelerated aging for the battery.
+uavcan.si.unit.electric_current.Scalar.1.0 charge_termination_treshold  # End-of-charging current threshold.
+uavcan.si.unit.voltage.Scalar.1.0          charge_voltage               # Total voltage of the resting battery pack after it reaching 100% SOC at 100% SOH (not per-cell).
```

Comment:
Not sure this is a good idea. The updated description is specific to a certain kind of Li-ion batteries only, whereas this type is meant to be generic. I would restore the original comment.

Reply:
Which of the two comments are you referring to ?

Reply:
```suggestion
uavcan.si.unit.voltage.Scalar.1.0          charge_voltage               # Total charging voltage (not per-cell).
```

Reply:
As a specification and descriptive document, the comment is too vague. There are many words used in the battery community that are vague and used colloquially . _Total charging voltage_ can mean the constant voltage at which battery should be charged,  or something else. 

This better grammatically?
```suggestion
uavcan.si.unit.voltage.Scalar.1.0          charge_voltage               # Total voltage of the resting battery pack at 100% SOC and 100% SOH (not per-cell).
```

Reply:
It's not the same thing though. The total resting pack voltage at 100% SoC/SoH is not the same as the charging voltage. The former is basically `design_cell_voltage_min_max[1]` times the number of cells. The latter is an imperative to the battery charger. Some chemistries (like lead-acid) require the charge voltage to be much higher than the total pack resting voltage. The comment is thus misleading. Paging @dimracer

Reply:
changed the comment for this